### PR TITLE
Add ShadowTranslate plugin

### PR DESCRIPTION
**Pull request by ADmad**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -287,6 +287,7 @@ Focus is on the specific task.
 ## I18n
 *Plugins for I18n (Internationalization) and L10n (Localization)*
 
+- [ShadowTranslate plugin](https://github.com/AD7six/cakephp-shadow-translate) - A plugin with shadow table based replacement for core's Translate behavior.
```
Comment (jadb) [inline README.md]: Alphabetical order ;)

Comment (ADmad) [inline README.md]: Doh :P

**Comment by ADmad** [comment]
Should be also remove Localized plugin from list? I doubt the datasource in there will be updated to 3.x. Even if they are they should be managed independently.

**Comment by dereuromark** [comment]
I hope @lorenzo shares his plans about localization and the plugin around it with us at the open ticket in the corresponding repo first ( https://github.com/cakephp/localized/issues/84 ).

**Comment by ADmad** [comment]
Doh, I said Localized above but I meant Datasources plugins :stuck_out_tongue:

**Comment by dereuromark** [comment]
;) I don't mind having those in the list for visibility in case someone needs it and maybe uses this opportunity to upgrade it (or part of it at least) then.

**Pull request merged**